wherehows-common/src/test/java/wherehows/common/utils/JobsUtilTest.java

```diff
@@ -116,6 +120,27 @@ public void testGetEnabledJobsByType() throws IOException {
     Files.deleteIfExists(path2);
   }
 
+  @Test
+  public void testMultipleLinesSameProperties() throws IOException, ConfigurationException {
+    String dir = "/tmp/";
+    String propertyStr =
+        "var1=foo\n" + "var1=foo\n" + "var1=foo\n" + "var2=foo\n" + "var2=foo\n" + "var3=foo\n" + "var1=foo\n"
+            + "var4=foo\n";
+    String propertyStr1 = "var5=foo,bar,test,test1\n" + "var5=test2\n";
+    Properties props = new Properties();
+    Path path = createPropertiesFile(propertyStr + propertyStr1);
+
+    props = getJobConfigProperties(path.toFile());
+
+    Assert.assertNotEquals(props, null);
+    Assert.assertEquals(props.getProperty("var1", ""), "foo,foo,foo,foo");
+    Assert.assertEquals(props.getProperty("var2"), "foo,foo");
+    Assert.assertEquals(props.getProperty("var3"), props.getProperty("var4"));
```

Comment:
ssert.assertEquals(props.getProperty("var3"), "foo")
ssert.assertEquals(props.getProperty("var4"), "foo")

Reply:
Got it , verified all the literal values now.